website/docs/docs/core/connect-data-platform/teradata-setup.md

```diff
@@ -225,6 +230,11 @@ For using cross-DB macros, teradata-utils as a macro namespace will not be used,
 
   `last_day` in `teradata_utils`, unlike the corresponding macro in `dbt_utils`, doesn't support `quarter` datepart.
 
+## dbt - unit tests
+dbt-teradata 1.8.0 and above versions support unit tests, allowing validation of SQL modeling logic on a small set of static inputs before production. This enhancement facilitates test-driven development, improving developer efficiency and code reliability.
+
+More information on dbt unit tests feature can be found [here](https://docs.getdbt.com/docs/build/unit-tests).
+
```

Comment:
hi @tallamohan -- please consider wrapping this (and any other relevant sections in a `<VersionBlock>`! More info on [versioning blocks of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content)

Reply:
Hi @dataders , I have added VersionBlock like the below:

`<VersionBlock firstVersion="1.8.0">
    dbt-teradata 1.8.0 and above versions support unit tests, allowing validation of SQL modeling logic on a small set of static inputs before production. This enhancement facilitates test-driven development, improving developer efficiency and code reliability.
    More information on dbt unit tests feature can be found [here](https://docs.getdbt.com/docs/build/unit-tests).
</VersionBlock>`

But in the preview, It is showing up like this :
![Screenshot 2024-06-25 110402](https://github.com/dbt-labs/docs.getdbt.com/assets/6680479/e8f2212b-e0a1-457a-82f8-7dcfdecc1e30)

Can you please help me with this?

Thanks,
Mohan